internal/resources/providers/awslib/identity_provider.go

```diff
@@ -44,13 +47,18 @@ func (p IdentityProvider) GetIdentity(ctx context.Context, cfg aws.Config) (*clo
 	}
 
 	alias, err := p.getAccountAlias(ctx, cfg)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get aliases: %w", err)
+	var aliasPtr *string
+
+	if err == nil {
+		aliasPtr = &alias
+	} else {
+		p.Logger.Errorf("failed to get aliases: %w", err)
```

Comment:
maybe a warning instead? This does not seem severe enough to be classified as an error IMO, especially since we will begin tracking error rates in serverless.

Reply:
Will do!